src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java

```diff
@@ -582,20 +584,17 @@ public static void loadRecipes()
         //TODO: Figure out how to remove recipes, and override them. This relies on cpw to help.
         //For now this is only done one after mod init, I want to move this to ServerInit and re-do it many times.
         init();
+        ForgeRegistry<IRecipe> reg = (ForgeRegistry<IRecipe>)ForgeRegistries.RECIPES;
         if (DEBUG_LOAD_MINECRAFT)
-        {
-            Iterator<IRecipe> itr = GameData.getRecipeRegistry().iterator();
-            while(itr.hasNext())
-            {
-                itr.next();
-                itr.remove();
-            }
-        }
+            reg.clear();
+        else
+            GameData.revert(RegistryManager.FROZEN, GameData.RECIPES, false);
```

Comment:
As it sits this function is only called once and is incomplete. Not calling anything from a modpack or server folder. So honestly it doesn't matter in the slightest. I'll deal with this after merging.